components/prism-cmake.js

```diff
@@ -0,0 +1,8 @@
+    "boolean": /\b(ON|OFF)\b/,
+    "namespace": /\b(PROPERTIES|SHARED|PRIVATE|STATIC|PUBLIC|INTERFACE|TARGET_OBJECTS)\b/,
+    operator: /[-+]{1,2}|!=?|<{1,2}=?|>{1,2}=?|\->|:{1,2}|={1,2}|\^|~|%|&{1,2}|\|?\||\?|\*|\/|\b(and|and_eq|bitand|bitor|not|not_eq|or|or_eq|xor|xor_eq)\b/,
```

Comment:
`[+-]{1,2}`: Does that mean that `+-` is a valid operator?

Reply:
CMake has different ones. This entire line will be replaced.